packages/sanity-check/App.test.tsx

```diff
@@ -1,9 +0,0 @@
-import { render, screen } from '@testing-library/react';
```

Comment:
why?

Reply:
Because it is and old test and it is failing